### PR TITLE
Automated cherry pick of #1941: bump go to 1.17.9

### PR DESCRIPTION
**Pull request by doublek**
Cherry pick of #1941 on release-v1.26.

#1941: bump go to 1.17.9

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Pull request merged**
